src/BowlingScorecard.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+class BowlingScorecard {
+  constructor() {
+    this.total_score = 0;
+    this.currentFrame = 1;
+    this.currentFrameRolls = 0;
+    this.scorecard = { 1: [] }
+  }
+
+recordRoll(userInput) {
+  this._checkUserInput(userInput);
+  // WIP to end game if 10th frame score < 10 after two rolls
```

Comment:
I like the idea of commenting out things you'd like to add if you had more time.